src/breadcrumbs/breadcrumbs.scss

```diff
@@ -20,28 +20,27 @@
 
 @mixin iui-breadcrumbs-item {
   display: flex;
-  align-items: center;
-
-  > .iui-anchor {
-    display: inline-block;
-  }
+  line-height: $iui-component-height;
+  height: $iui-component-height;
+  max-width: $iui-3xl * 2;
+  margin: 0 $iui-sm;
 
-  > .iui-anchor,
-  > span,
-  .iui-ellipsis {
-    margin: 0 $iui-sm;
+  > .iui-button {
+    margin: 0 (-$iui-s);
   }
 
-  > .iui-button {
-    margin: 0 $iui-xs;
+  &:last-child {
+    &,
+    > .iui-button {
+      margin-right: 0;
```

Comment:
Does it make any difference?

Reply:
It increases the width, but looks same visually. I could remove

Reply:
I dont see the width change with or without it

Reply:
well yeah, i said "looks same visually" 😁

I've removed and pushed. See the change in test images